.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Issue Report
+about: Create a report to help us improve
+title: Issue Report
+labels: ''
+assignees: ''
+
+---
+
+**Issue Overview**
+Brief overview of the issue
```

Comment:
You can wrap these lines in comment tags like `<!-- blahblahblah -->` so that they are visible when filling out the issue but not once posted! You might have seen them when posting PRs in the app repo